### PR TITLE
New C# Wrapper

### PR DESCRIPTION
**Pull request by siavashserver**
Addresses #47 .

### CODE REVIEW

File: _posts/2018-06-25-CSharp-wrapper-released.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: post
+title: C# wrapper released
+tags: news
+---
+
+Eric Freed has released a complete, cross-platform, managed wrapper called 
+[glfw-net](https://github.com/ForeverZer0/glfw-net) which covers GLFW 3.2 
+library for creating native windows with an OpenGL context.
```
Comment (elmindreda) [inline _posts/2018-06-25-CSharp-wrapper-released.md]: This last line breaks the pattern from previous similar posts. Should we do it this way from now on?

Comment (siavashserver) [reply _posts/2018-06-25-CSharp-wrapper-released.md]: My bad, didn't notice the pattern. I have no issues if you would like to keep consistent with previous posts.

Comment (siavashserver) [reply _posts/2018-06-25-CSharp-wrapper-released.md]: @elmindreda Is this pull request okay as it is? or do you prefer it to follow the same pattern from previous posts?

Comment (elmindreda) [reply _posts/2018-06-25-CSharp-wrapper-released.md]: I'd prefer to stick to the previous pattern.

**Comment by siavashserver** [comment]
It's ready to be squashed and merged :-)

**Comment by elmindreda** [comment]
Oops, missed this issue pair somehow. Sorry! Waiting for reply in #47.

**Comment by elmindreda** [comment]
Merged and updated. Thank you!

**Pull request closed without merging**